I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 416, in script call
    call prologue
  File "game/script.rpy", line 52, in script
    $show_text_slowly(dot_mozi("あいうろ。いいね"))
  File "game/script.rpy", line 52, in <module>
    $show_text_slowly(dot_mozi("あいうろ。いいね"))
  File "game/script.rpy", line 17, in show_text_slowly
    renpy.say(char)
TypeError: say() missing 1 required positional argument: 'what'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 416, in script call
    call prologue
  File "game/script.rpy", line 52, in script
    $show_text_slowly(dot_mozi("あいうろ。いいね"))
  File "C:\Users\daruma\use-bin\renpy\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\daruma\use-bin\renpy\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 52, in <module>
    $show_text_slowly(dot_mozi("あいうろ。いいね"))
  File "game/script.rpy", line 17, in show_text_slowly
    renpy.say(char)
TypeError: say() missing 1 required positional argument: 'what'

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
gizin_kaden 1.0
Mon Jan 29 07:25:14 2024
